scrapers/AdultEmpire.yml

```
@@ -102,4 +108,31 @@ xPathScrapers:
                 - regex: ^
                   with: "https://www.adultdvdempire.com"
       URL: //meta[@name='og:url']/@content
-# Last Updated October 09, 2023
+  performerScraper:
+    common:
+      $infoPiece: //*[@id="profileModal"]/div/div/div[2]/div[1]/ul
+    performer:
+      Name: //*[@id="content"]/section/div/div[2]/h1/text()
+      Birthdate: $infoPiecel/li[contains(text(), 'Born:')]/text()
+      Gender:
+        fixed: "Female"
+      Height:
+        selector: $infoPiecel/li[contains(text(), 'Height:')]/text()
+        postProcess:
+          - replace:
+              - regex: ^(\d+).*
+                with: $1
+      Image: //*[@id="content"]/section/div/div[1]/a[1]/img/@src
+      Country: $infoPiecel/li[contains(text(), 'From:')]/text()
```

Same typo, same procedure
```suggestion
      Country:
        selector: $infoPiecel/li[contains(text(), 'From:')]/text()
        postProcess:
          - replace:
              - regex: From:\s+(.*)
                with: $1
```

---

This still won't give the correct country, but that's a limitation of their website: they list this as "city, state" for American performers and "city, country" for everyone else

Still better than nothing 🙂